compose-app/src/main/java/com/mapbox/maps/compose/MainActivity.kt

```diff
@@ -3,57 +3,53 @@ package com.mapbox.maps.compose
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.viewinterop.AndroidView
 import com.mapbox.geojson.Point
-import com.mapbox.maps.CameraOptions
-import com.mapbox.maps.MapView
+import com.mapbox.maps.MapInitOptions
 import com.mapbox.maps.Style
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.extension.compose.MapboxMap
 
-private const val LATITUDE = 60.239
-private const val LONGITUDE = 25.004
-
-class MainActivity : ComponentActivity() {
+/**
+ * Example to showcase usage of MapView
```

Comment:
```suggestion
 * Example to showcase usage of MapboxMap
```
since it's called MapboxMap now?